casters-and-coders-prototype/interface/theme/theme.tres

```diff
@@ -8,7 +8,6 @@
 [ext_resource path="res://interface/theme/stylebox_green_pixel_pressed.tres" type="StyleBox" id=6]
 
 [sub_resource type="StreamTexture" id=2]
-load_path = "res://.import/panel.png-9535982eaded27d3e12fceba743de959.stex"
```

Comment:
This is a result of #16 and merging this will cause the same issue for Brennan. Let's revert this change for now and we can fix the issue in another PR